src/exercises/Congratulations.js

```diff
@@ -154,25 +102,12 @@ export default function Congratulations({
               </b>
             )}
           </p>
-
-          {isOverTotalBookmarkLimit && (
-            <p>
-              You have already <b>{totalBookmarksInPipeline} words</b> you are
-              learning at the moment. We recommend that you at{" "}
-              <b>most learn {MAX_EXERCISE_IN_LEARNING_BOOKMARKS} words</b> at
-              any given point.
-            </p>
-          )}
-          {canStartLearningNewWords && !isOverTotalBookmarkLimit && (
-            <p>
-              You can start <b>studying new words</b>, do you want to continue
-              exercising?
-            </p>
-          )}
           {!isThereMoreExercises && (
             <p>
               There are no more words for you to practice. You can read more
-              articles and find new words to learn!
+              articles and find new words to learn! We will let you know when
+              it's time to review your words according to our spaced-repetition
+              schedule.
```

Comment:
I love that we're dropping the spaced repetition in here. We need to do it more ;)